middlewares/taskRequests.js

```diff
@@ -1,12 +1,17 @@
 const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
 const dataAccess = require("../services/dataAccessLayer");
+const { TASK_REQUEST_UPDATE_ACTION } = require("../constants/taskRequests");
 /**
  * Validates user id for task request
  *
  * @param userId { string }: user id of the user
  */
 async function validateUser(req, res, next) {
   try {
+    const { action } = req.query;
+    if (action === TASK_REQUEST_UPDATE_ACTION.REJECT) {
+      return next();
```

Comment:
What about ACCEPT?

Reply:
- Validation will happen for the provided user id.
- UserId is not required for rejecting a task request. So no validation is required.